library/ssl_tls.c

```diff
@@ -6002,9 +6011,10 @@ static int ssl_buffer_message( mbedtls_ssl_context *ssl )
 
                 /* Prepare final header: copy msg_type, length and message_seq,
                  * then add standardised fragment_offset and fragment_length */
-                mbedtls_platform_memcpy( hs_buf->data, ssl->in_msg, 6 );
+                /* Not using more secure mbedtls_platform_memcpy as final header is not that critical and small buffers */
```

Comment:
Also public (transmitted in the clear in the initial handshake).

Reply:
fixed